Mesh_3/test/Mesh_3/CMakeLists.txt

```diff
@@ -121,6 +121,21 @@ if ( CGAL_FOUND )
         target_link_libraries(${target} PUBLIC CGAL::TBB_support)
       endif()
     endforeach()
+
+    if(BUILD_TESTING)
+      set_property(TEST execution___of__test_meshing_verbose PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_polyhedron_with_features PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_implicit_function PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_3D_image PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_3D_gray_image PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_unit_tetrahedron PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_polyhedron PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_meshing_polyhedral_complex PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_mesh_capsule_var_distance_bound PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_mesh_3_issue_1554 PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_mesh_polyhedral_domain_with_features_deprecated PROPERTY RUN_SERIAL 1)
+      set_property(TEST execution___of__test_mesh_cell_base_3 PROPERTY RUN_SERIAL 1)
```

Comment:
That is just a matter of style, and not something I ask you to change: you can specify multiple test names in one command:
https://cmake.org/cmake/help/latest/command/set_property.html
> `TEST`
> Scope may name zero or more existing tests.

Reply:
You are right! I missed that.